spiel/st/spiel_multiple_confd.sh

```diff
@@ -155,11 +155,11 @@ confd_mkfs_start() {
     -m $MAX_RPC_MSG_SIZE -q $TM_MIN_RECV_QUEUE_LEN -c $CONF_FILE\
     -w 3 -f ${!fid}"
 
-    echo "--- `date` ---" >>$path/m0d.log
-    cd $path
+    echo "--- `date` ---" >>"$path"/m0d.log
+    cd "$path"
 
-    echo $M0_SRC_DIR/utils/mkfs/m0mkfs $OPTS
-    $M0_SRC_DIR/utils/mkfs/m0mkfs $OPTS >>$path/mkfs.log ||
+    echo "$M0_SRC_DIR"/utils/mkfs/m0mkfs $OPTS
+    "$M0_SRC_DIR"/utils/mkfs/m0mkfs "$OPTS" >>"$path"/mkfs.log ||
```

Comment:
Remove the double quote around $OPTS
It is error, I think.

And the same for Line 180 and elsewhere.

Reply:
done